static/app/views/issueDetails/groupSimilarIssues/similarStackTrace/index.tsx

```diff
@@ -47,6 +47,9 @@ function SimilarStackTrace({params, location, project}: Props) {
   const navigate = useNavigate();
   const prevLocationSearch = usePrevious(location.search);
   const hasSimilarityFeature = project.features.includes('similarity-view');
+  const hasSimilarityEmbeddingsFeature = project.organization?.features?.includes(
```

Comment:
is there another way to get organization if the `project.organization` does not exist?